cypress/e2e/Taxdata/Documentation.cy.js

```diff
@@ -0,0 +1,59 @@
+describe('Taxdata documentation page', () => {
+
+    Cypress.on('uncaught:exception', (err, runnable) => {
+        // Returning false here prevents Cypress from failing the test
+        return false;
+    });
+
+    before(() => {
+        cy.visit('https://apilayer.com/marketplace/tax_data-api');
+    });
+
+    it('0. Test to Accept Cookies if not already accepted', () => {
+        cy.AcceptCookies();
+    });
+
+    it('1. Test to check for the "Documentation" tab present or not', () => {
+        cy.componentVisiblityCheck('.d-md-flex > .d-none');
+    });
+
+    it('2. Test to click on the "Documentation" tab', () => {
+        cy.get('.d-md-flex > .d-none').click();
+        cy.wait(5000);
```

Comment:
replace with the code discussed using the readystate event.

Reply:
Not worked with given code, added the Todo for this.